assets/src/edit-story/elements/utils/duplicatePage.js

```diff
@@ -25,7 +25,14 @@ import { v4 as uuidv4 } from 'uuid';
 import createNewElement from './createNewElement';
 
 const duplicatePage = (oldPage) => {
-  const { elements: oldElements, animations: oldAnimations, ...rest } = oldPage;
+  // Remove title and postId for inserting the page.
+  const {
+    elements: oldElements,
+    animations: oldAnimations,
+    postId,
+    title,
+    ...rest
+  } = oldPage;
```

Comment:
Hm, has this been a problem previously? Also, can't we just use the `objectWithout` util?

Reply:
It wasn't a problem before but it is with the saved templates now -- if the user adds a template that has a title and then creates a custom template based on that, the title would otherwise also be included and this causes the saved templates to have a visible title on hovering which is unexpected.

On using `objectWithout` -- we could, but that util is more complex as opposed to just extracting the title like this. Are you thinking of readability?

Reply:
> if the user adds a template that has a title and then creates a custom template based on that, the title would otherwise also be included and this causes the saved templates to have a visible title on hovering which is unexpected.

Ah, that's a rather specific usecase, but probably makes sense - people will make their own variants of the defaults!

> On using `objectWithout` -- we could, but that util is more complex as opposed to just extracting the title like this. Are you thinking of readability?

Yeah, I know there's a comment, but multiple things are happening at the same time. Also doesn't it cause unused variables? I'd split it up in several lines like this so you know what's happening coming back to it in 6 months:

```js
  const {
    elements: oldElements,
    animations: oldAnimations,
    ...rest
  } = oldPage;
  const cleanPage = objectWithout(rest, ['postId', 'title']);
```